css/components/modal.css

```diff
@@ -0,0 +1,62 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@400;500;600;700;800&display=swap");
+@import url("../utility.css");
+@import url("./button.css");
+
+.modal {
+    max-width: 70%;
```

Comment:
Just a suggestion: can also use clamp here
`width: clamp(14rem, 70%, any_max_width);`

Reply:
Will look into it